app/controllers/donation_sites_controller.rb

```diff
@@ -4,8 +4,11 @@ class DonationSitesController < ApplicationController
   include Importable
 
   def index
-    @donation_sites = current_organization.donation_sites.active.alphabetized
-    @donation_site = current_organization.donation_sites.new
+    @donation_sites = current_organization.donation_sites.alphabetized
+    @donation_sites = @donation_sites.active if params[:include_inactive_donation_sites].blank?
+
+    @include_inactive_donation_sites = params[:include_inactive_donation_sites]
```

Comment:
Minor, but you should swap these around to use the name since you're defining it anyway.

```suggestion
    @include_inactive_donation_sites = params[:include_inactive_donation_sites]
    @donation_sites = @donation_sites.active if @include_inactive_donation_sites.blank?
```